### PR TITLE
Matched name of pipeline to SCF/UFR target/script names.

### PR DESCRIPTION
**Pull request by andreas-kupries**
https://trello.com/c/eX0agPsP/189-clean-up-the-names-of-makefile-targets-and-scripts
Related PRs
- https://github.com/SUSE/scf/pull/1006
- https://github.com/SUSE/uaa-fissile-release/pull/25

### CODE REVIEW

File: cf-kube-dist/cf-kube-dist.yml [hunk 1]
```diff
@@ -6,15 +6,15 @@ resources:
   source:
     branch: develop
     private_key: {{github-private-key}}
```
Comment (mook-as) [inline cf-kube-dist/cf-kube-dist.yml]: That means we no longer need the private key, right? (Since we're not using ssh anymore)

**Comment by andreas-kupries** [comment]
Looks like I missed a `kube-config`
See also https://github.com/SUSE/uaa-fissile-release/pull/24
in this set.

File: cf-kube-dist/cf-kube-dist.yml (hunk 1 above)
Comment (andreas-kupries) [inline cf-kube-dist/cf-kube-dist.yml]: I guess so ?

**Comment by andreas-kupries** [comment]
Addressed in commit 7358fe4d50a3d5206cf2f2e416cc51bc6d52df64

File: cf-kube-dist/cf-kube-dist.yml (hunk 1 above)
Comment (andreas-kupries) [inline cf-kube-dist/cf-kube-dist.yml]: Addressed in commit c4b6035602bb93e45fcb73f18f38db346a118220

**Comment by andreas-kupries** [comment]
Should be fixed with commit 0c9e2697f6adc1ebc608105962a30694e1176138

**Pull request merged**